libraries/BLE/examples/BLE_EddystoneTLM_Beacon/BLE_EddystoneTLM_Beacon.ino

```diff
@@ -48,10 +49,10 @@ void setBeacon()
   char beacon_data[25];
   uint16_t beconUUID = 0xFEAA;
   uint16_t volt = random(2800, 3700); // 3300mV = 3.3V
-  float tempFloat = random(2000, 3100) / 100.0f;
-  Serial.printf("Random temperature is %.2fC\n", tempFloat);
-  int temp = (int)(tempFloat * 256); //(uint16_t)((float)23.00);
-  Serial.printf("Converted to 8.8 format %0X%0X\n", (temp >> 8), (temp & 0xFF));
+  float tempFloat = random(-3000, 3000) / 100.0f;
+  Serial.printf("Random temperature is %.2f°C\n", tempFloat);
+  int temp = EDDYSTONE_TEMP_FLOAT_TO_U16(tempFloat);
+  Serial.printf("Converted to 8.8 format 0x%04X\n", temp);
```

Comment:
The original code was working correclty. There is no need to change it.
Please revert the change in lines 51 to 55.

I tested the proposed new code and it doesn't work.

The problem with the new code is that `EDDYSTONE_TEMP_FLOAT_TO_U16()` returns a `big endian uint16_t` and the original code was using `temp` as `little endian` which was turned into `big endian` with the lines: 
``` cpp
  beacon_data[4] = (temp >> 8);           // Beacon temperature
  beacon_data[5] = (temp & 0xFF);           //

```

Reply:
Please revert to:
``` cpp
float tempFloat = random(2000, 3100) / 100.0f;
  Serial.printf("Random temperature is %.2fC\n", tempFloat);
  int temp = (int)(tempFloat * 256); //(uint16_t)((float)23.00);
  Serial.printf("Converted to 8.8 format %0X%0X\n", (temp >> 8), (temp & 0xFF));
```